util/stats/query_stats.go

```diff
@@ -144,14 +162,52 @@ func (s *SpanTimer) Finish() {
 	}
 }
 
+type Statistics struct {
+	Timers  *QueryTimers
```

Comment:
These don't need to be references, which will save a bit of CPU for GC.

Reply:
Oh yes. I guess, this was done for some other purpose where structs were long, but over the due course of time, that was changed leaving this.